Source/DataSources/PolylineGeometryUpdater.js

```diff
@@ -446,12 +453,8 @@ define([
         this._primitives = primitives;
         this._geometryUpdater = geometryUpdater;
         this._positions = [];
-    };
 
-    var generateCartesianArcOptions = {
-        positions : undefined,
-        granularity : undefined,
-        height : undefined
+        generateCartesianArcOptions.ellipsoid = geometryUpdater._scene.globe.ellipsoid;
```

Comment:
@emackey, can you submit a separate issue for this.  A quick search for `scene.globe.` shows that there are quite a few places we are access a property on globe without checking.  For the specific case of `scene.globe.ellipsoid`, we should probably add an ES5 getter for `scene.ellipsoid` so that this mistake is harder to make.